library/src/skins/default/src/sass/modules/tool/portal/_portal.scss

```diff
@@ -56,6 +56,22 @@
   grid-area: header;
   box-shadow: 0 3px 1px -2px rgba(0, 0, 0, 0.2), 0 2px 2px 0 rgba(0, 0, 0, 0.14), 0 1px 5px 0 rgba(0, 0, 0, 0.12);
 }
+
+// Responsive header for mobile
+@media (max-width: 767px) {
+  .portal-header {
+    background-color: rgba(var(--bs-primary-rgb), var(--bs-bg-opacity)) !important;
+    justify-content: space-between !important;
```

Comment:
they are adding the rules, but we need to override them for smaller devices Because the header has a different layout on small devices compared to the desktop version.